src/store/modules/favoriteSpaces.ts

```diff
@@ -0,0 +1,41 @@
+import Vue from 'vue';
+import omit from 'lodash/omit';
+import { lsGet, lsSet } from '@/helpers/utils';
+
+const FAVORITES_KEY = 'FAVORITE_SPACES';
+
+const state = {
+  favorites: {}
+};
+
+const mutations = {
+  setFavorites(_state, favorites: Record<string, boolean>) {
+    Vue.set(_state, 'favorites', favorites);
+  }
+};
+
+const actions = {
+  loadFavoriteSpaces({ commit }) {
+    const favorites = lsGet(FAVORITES_KEY);
+
+    commit('setFavorites', favorites);
+  },
+  addFavoriteSpace({ commit, state }, spaceId: string) {
+    const favorites = { ...state.favorites, [spaceId]: true };
```

Comment:
Any reason to use this format `{ space1: true, space2: true }` instead of` ['space1', 'space2']`?

Reply:
Mostly to speed up lookups. Checking if space if favourited or not with `includes` is more expensive than just object lookups, but the performance shouldn't be an issue in this case.
This also saves us from deduping on adding new entry.